File: JAVA/Daily Flash Codes/Week2/Day3_Jan22/code/prog1.java
import java.io.*;
class Program {
	public static void main(String args[]) throws IOException{

		BufferedReader br = new BufferedReader (new InputStreamReader(System.in));
		System.out.println("Enter two nums: ");
		int num1 = Integer.parseInt(br.readLine());
		int num2 = Integer.parseInt(br.readLine());
		
		System.out.println("Addition : "+ (num1+num2));

		if(num1 > num2)	
			System.out.println("Substraction: "+ (num1 - num2));
		else
			System.out.println("Substraction : "+ (num2 - num1));
		

	}
}

File: C/Daily Flash/Week2/Day3_Jan22/code/prog1.c
#include<stdio.h>

void main() {

	int num1,num2;
	printf("Enter number 1: ");
	scanf("%d",&num1);

	printf("Enter number 2: ");
	scanf("%d",&num2);
	
	//addtion
	printf("Addition = %d\n",num1+num2);

	//sub
	if(num1 > num2)
		printf("Substraction = %d\n",num1 - num2);
	else
		printf("Substraction = %d\n",num2 - num1);
}


